dev_mode/webpack.config.js

```diff
@@ -273,9 +273,9 @@ module.exports = [
       poll: 500,
       aggregateTimeout: 1000
     },
-    node: {
-      fs: 'empty'
-    },
+    // node: {
+    //   fs: 'empty'
+    // },
     bail: true,
     devtool: 'inline-source-map',
     externals: ['node-fetch', 'ws'],
```

Comment:
In the next release (beta 17) this will accept promises, function strings, and http URLs:)

Reply:
Sweet!